packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/model/Model.java

```diff
@@ -32,6 +32,8 @@ public class Model extends TerariumAsset implements SupportAdditionalProperties,
 
 	private Map<String, Object> model;
 
+	private String modelCard;
```

Comment:
Not sure where the model card is supposed to be written. This is probably not correct.